analysis/monk/src/SinisterTeachings.tsx

```diff
@@ -13,7 +14,8 @@ import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
 import * as React from 'react';
 
 const ICD = 750;
-const CDR = 5000;
+const CDR_DEFAULT = 5000;
```

Comment:
A more accurate variable name would be `CR_FOR_LOSER_SPECS_THAT_ARE_NOT_MISTWEAVER_HAHAHA_SUCKS_TO_BE_WW_OR_BREW`